test/documentation/CMakeLists.txt

```diff
@@ -23,7 +23,8 @@ enable_testing()
 set (SEQAN3_DOXYGEN_TEST_SCRIPT "
     output=\$(${DOXYGEN_EXECUTABLE} 2>&1 > /dev/null | sort);
     echo \"\$output\";
-    test -z \"\$output\"")
+    test -z \"\$output\";
+    ! find . -not -name \"*_source.html\" -name \"*.html\" -print0 | xargs -0 grep \"requires\" | grep \"memname\"")
```

Comment:
```suggestion
    !find . -not -name \"*_source.html\" -name \"*.html\" -print0 | xargs -0 grep \"requires\" | grep \"memname\"")
```
?

Reply:
No, that's invalid. At least Ubuntu doesn't like it.

Reply:
Wow.